pkg/web/api/v1/transaction.go

```diff
@@ -226,6 +228,15 @@ func (c *Transaction) Model() (model *models.Transaction, err error) {
 	return model, nil
 }
 
+func (e *Envelope) EnvelopeDump() string {
```

Comment:
```suggestion
func (e *Envelope) Dump() string {
```

`Envelope.EnvelopeDump` "stutters" so Go style guides prefer simply `Envelope.Dump`.